autoload/android.vim

```diff
@@ -314,7 +314,8 @@ function! android#emulator()
   let l:option = l:avds[l:choice]
   let l:avd = strpart(l:option, 3)
 
-  execute 'silent !' . android#emulatorbin() . ' -avd ' . l:avd . ' 2>/dev/null  &'
+  let l:options = {}
```

Comment:
Can you try adding detach option? this way this will behave same way as if using execute.

```suggestion
  let l:options = { detach: 1 }
```